pkg/reconciler/instances/istio/action.go

```diff
@@ -201,6 +201,16 @@ func canUninstall(istioVersion actions.IstioVersion) bool {
 	return isInstalled(istioVersion) && istioVersion.ClientVersion != ""
 }
 
+func (h *helperVersion) isDowngradePermitted(second *helperVersion) bool {
+	// we permit one minor downgrade, please see: https://github.com/kyma-incubator/reconciler/issues/365
```

Comment:
I find such reasoning as unnecessary explanations. In addition, we should not provide comments next to the code (we have documentation for it and this one is not aligned with clean code rules).

Reply:
I honestly don't see a problem with documentation and I'm always used to commenting when necessary. If we follow the pattern in the team to not comment, I will take it off.